specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1652,6 +1652,90 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityInsights/enrichment/ip/geodata/": {
```

Comment:
>ip/geodata [](start = 120, length = 10)

ip address in query string instead of /ip/xxx/geodata?

Reply:
Hi @allenjzhang I actually had ip as a path parameter originally but changed it at the suggestion of the original ARM reviewers (this was in an email conversation not captured in my private repo PR).   My original PR has already received ARM approval and an exception for the API version.  (link at the top of this PR)

Reply:
@allenjzhang Can you resolve this and approve/complete the PR?